hack/ci/Vagrantfile-flatcar

```diff
@@ -0,0 +1,118 @@
+#-*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "flatcar-stable"
+  config.vm.box_url = "https://stable.release.flatcar-linux.net/amd64-usr/current/flatcar_production_vagrant.json"
+  memory = 8192
+  cpus = 4 
+
+  config.vm.provider :virtualbox do |v|
+     v.customize [
+      "modifyvm", :id,
+      "--memory", memory,
+      "--paravirtprovider", "default",
+      "--cpus", cpus 
+    ]
+    v.check_guest_additions = false
+    v.functional_vboxsf = false
+  end
+
+  config.vm.provider :libvirt do |v|
+    v.memory = memory
+    v.cpus = cpus
+  end
+
+  config.ssh.username = 'core'
+  config.ssh.insert_key = true
+  config.ssh.forward_agent = true
+
+  config.vm.synced_folder ".", "/vagrant", type: "rsync"
+
+  config.vm.provision "install-dependencies", type: "shell", run: "once" do |sh|
+    sh.inline = <<~SHELL
+      set -euxo pipefail
+
+      # Install Go
+      GO_VERSION=1.18
+      mkdir /opt/go
+      curl -sSfL -o- https://golang.org/dl/go$GO_VERSION.linux-amd64.tar.gz | tar xfz - -C /opt
+
+      # Setup Networking 
+      systemctl enable docker
```

Comment:
Why do we need to enable Docker? Isn't containerd enough?

Reply:
I'll switch if off and try again only with containerd. Let's see if it works.